packages/cli/src/util/build/write-build-result.ts

```diff
@@ -156,11 +159,18 @@ async function writeBuildResultV2(
         const ext = getFileExtension(fallback);
         const fallbackName = `${normalizedPath}.prerender-fallback${ext}`;
         const fallbackPath = join(outputDir, 'functions', fallbackName);
-        const stream = fallback.toStream();
-        await pipe(
-          stream,
-          fs.createWriteStream(fallbackPath, { mode: fallback.mode })
-        );
+
+        // if we're in the build container and fallback is already
+        // on the disk just move it to avoid duplicating output
```

Comment:
The comment is no longer true since its not moving, right?

Also, can we do this every time (not just build container)?

Reply:
I think hardlink should be safe always actually, only was branching since move could have potentially caused issues if the build files were used separate of `vc build` locally

Reply:
Updated to fallback to copy on link failure